accepted/PSR-7-http-message.md

```diff
@@ -1903,3 +1903,6 @@ interface UploadedFileInterface
     public function getClientMediaType();
 }
 ```
+
+Since [psr/http-message version 1.1](https://packagist.org/packages/psr/http-message#1.1.0), the above interfaces have been updated to add argument type hints.
```

Comment:
It should not be called "type hint", because it's not just a "hint", the type is actually enforced.

Reply:
To make sure it isn't missed: The same applies in the line below, for the return types.

Reply:
Updated both to use the verbiage "type declarations" with 2b99fb2.